R/maic_anchored.R

```diff
@@ -485,8 +485,8 @@ maic_anchored_binary <- function(res,
   # ~~~ Analysis table
   # : set up proper link
   glm_link <- switch(eff_measure,
-    "RD" = poisson(link = "identity"),
-    "RR" = poisson(link = "log"),
+    "RD" = binomial(link = "identity"),
```

Comment:
@MikeJSeo From the checks I see that `binomial("identity")` is not valid.

Reply:
@gravesti 
so binomial("identity") does seem to work when the covariate is discrete (0/1 like as in treatment indicator), but doesn't work if the covariate is continuous. Would this possibly be the reason?
Which check does it fail?

Or could this be R version issue?
https://stats.stackexchange.com/questions/139917/r-binomial-family-with-identity-link

It does seem to change the results quite a bit

Reply:
Looking in the code for `glm` and `binomial` I don't see why there should be a warning with out code. The link function is correctly set as identity, so let's leave that for now.

Regarding the changed results:

- I think the fix to the N makes the `$descriptive` table correct.

- The changes to the SE (and p-val) I'm not sure. I'd be pleased if @hoppanda could review...

Reply:
Hi @gravesti,
This originally started from my mistake where I thought this identity or log link doesn't work for binomial.
It turns out that for binary covariates, this works fine.
Can you merge this in for the time being? I think this is quite a big of an error that we don't want people to use in terms of SE/p-val.